ql/instruments/floatfloatswaption.cpp

```diff
@@ -47,12 +49,32 @@ namespace QuantLib {
 
         arguments->swap = swap_;
         arguments->exercise = exercise_;
+        arguments->settlementType = settlementType_;
+        if(settlementMethod_) {
+            arguments->settlementMethod = *settlementMethod_;
+        }
+        else {
+            // if the settlement method is not given, default
+            // values are used
+            arguments->settlementMethod =
```

Comment:
This appears a few times. It looks like it could be a static `Settlement::defaultMethod(Type)` method...

Reply:
agreed